fire/SFMainMod.F90

```diff
@@ -288,306 +344,126 @@ subroutine CalculateSurfaceRateOfSpread(currentSite)
   end subroutine CalculateSurfaceRateOfSpread
   
   !---------------------------------------------------------------------------------------
+  
+  subroutine CalculateSurfaceFireIntensity(currentSite)
+    !
+    !  DESCRIPTION:
+    !  Calculates surface fireline intensity for each patch of a site
+    !  Right now also calculates the area burnt...
+    !
+    use FatesConstantsMod, only : m2_per_km2
+    use SFEquationsMod,    only : FireDuration, LengthToBreadth
+    use SFEquationsMod,    only : AreaBurnt, FireSize, FireIntensity
```

Comment:
`m2_per_km2`, `FireDuration`, `LengthToBreadth`, `AreaBurnt`, and `FireSize` are no longer used.